cmd/oras/root/manifest/push.go

```diff
@@ -184,7 +185,7 @@ func pushManifest(cmd *cobra.Command, opts pushOptions) error {
 		}
 		return opts.Output(os.Stdout, descJSON)
 	}
-	status.Print("Pushed", opts.AnnotatedReference())
+	printer.Print("Pushed", opts.AnnotatedReference())
```

Comment:
@TerryHowe Is this blocking your implementation of ORAS Golang scripting? I think this output should be classified to metadata output and it's better to 
```golang
_, displayMetadata := display.NewPushHandler("", nil, opts.Verbose) // avoiding formatting and TTY output
displayMetadata.OnCopied(&opts.Target)
```

Reply:
What I'm trying to develop here is a layer similar to logrus that doesn't know what is is printing, it just knows how to print. To me at least, ideally the format code would be a layer on top of that.

Reply:
Gotcha. Any good idea to inject the writer in? The writer comes from stdout or cobra command out.